homeassistant/components/iammeter/sensor.py

```diff
@@ -107,24 +110,10 @@ def unit_of_measurement(self):
         """Return the unit of measurement."""
         return self.unit
 
-    @property
-    def should_poll(self):
-        """Poll needed."""
-        return False
-
-    @property
-    def available(self):
-        """Return if entity is available."""
-        return self.coordinator.last_update_success
-
     async def async_added_to_hass(self):
         """When entity is added to hass."""
         self.coordinator.async_add_listener(self.async_write_ha_state)
 
     async def async_will_remove_from_hass(self):
         """When entity will be removed from hass."""
         self.coordinator.async_remove_listener(self.async_write_ha_state)
```

Comment:
I think these can be removed as well

Reply:
I am not sure about removing `async_will_remove_from_hass` since it is not implemented in `CoordinatorEntity`.
However, I think I can remove `async_added_to_hass` since it is implemented in `CoordinatorEntity`.

Reply:
The `async_will_remove_from_hass` is safe to remove here. It is the old way of doing what `async_on_remove` is now doing.

Reply:
Oh okay thanks for the tip :+1: